config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "playerBrandingId" : "e48c9b51282d406f957c666149763424",
```

Comment:
Whose playerID is this? This is a sample receiver, therefore customer need to replace this with their own playerID.

Perhaps we need to put a comment here?